src/main/java/com/sourceauditor/spdx_to_osv/Main.java

```diff
@@ -272,7 +272,10 @@ public static void spdxToOsv(IModelStore fromStore, String documentUri, Writer w
                 Optional<String> packageName = pkg.getName();
                 Optional<String> version = pkg.getVersionInfo();
                 if (packageName.isPresent() && version.isPresent()) {
-                    pvSet.add(new OsvVulnerabilityRequest(new OsvPackage(packageName.get(), "OSS-Fuzz", null),
+                    String pName = packageName.get();
+                    // Trim version info at end of name if it exists
+                    pName = pName.split("@")[0];
+                    pvSet.add(new OsvVulnerabilityRequest(new OsvPackage(pName, null, null),
```

Comment:
IMHO I think the creators of the SPDX documents should parse the version out and put it in the version field.

However, since we know at least one major generator of docs does this, I think we should leave the code in.  Perhaps we should just add to the comment something like `Some SPDX documents are created with the package name including the versions.  Although these should be parsed by the creator, this code will workaround the package names`.

Feel free to modify / shorten the comment suggested above

Reply:
Agreed - will add in that comment.